.github/workflows/tests.yml

```diff
@@ -0,0 +1,100 @@
+on: [push, pull_request]
+
+name: tests
+
+jobs:
+  install_pcap:
+    name: Install libpcap-dev
+    runs-on: ubuntu-latest
+    steps:
+      - run: sudo apt install libpcap-dev
+
+  check:
```

Comment:
Is there a reason for doing cargo check? We build it anyway.

Reply:
Yeah I guess you're right. I copy pasted from some template.